src/axom/mir/examples/tutorial_simple/runMIR.hpp

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+#ifndef AXOM_MIR_EXAMPLES_TUTORIAL_SIMPLE_RUNMIR_HPP
+#define AXOM_MIR_EXAMPLES_TUTORIAL_SIMPLE_RUNMIR_HPP
+#include "axom/config.hpp"
+#include "axom/core.hpp"  // for axom macros
+#include "axom/slic.hpp"
+#include "axom/mir.hpp"  // for Mir classes & functions
+
+#include <conduit/conduit_node.hpp>
+
+//--------------------------------------------------------------------------------
+/*!
+ * \brief Run MIR on the tri input mesh.
```

Comment:
Observation -- there are functions for `tri`, `quad` and `hex`, but not `tet`.

Reply:
tet is not in there for now in the tutorial_simple program because the current MeshTester does not emit tet zones. I added a comment to note that in the program.